lib/makeup/lexers/elixir_lexer.ex

```diff
@@ -25,7 +25,7 @@ defmodule Makeup.Lexers.ElixirLexer do
   # TODO: check we're following this convention
   # NOTE: if Elixir had a good static type system it would help us do the right thing here.
 
-  whitespace = ascii_string([?\r, ?\s, ?\n, ?\f], min: 1) |> token(:whitespace)
+  whitespace = ascii_string([?\r, ?\s, ?\n, ?\f, ?\t], min: 1) |> token(:whitespace)
```

Comment:
We don’t allow \f as white space though, so we can remove it.

```suggestion
  whitespace = ascii_string([?\r, ?\s, ?\n, ?\t], min: 1) |> token(:whitespace)
```

Reply:
Sure can remove it. What about Erlang? Both already supported `?/f`

Reply:
I checked \f in Erlang before merging there and it worked!

Reply:
Ok removed

Reply:
Sorry commit message messed up as I keep forgetting I can't use backticks in commit msgs on the command line, please squash